packages/automerge-repo/test/Repo.test.ts

```diff
@@ -1516,9 +1471,9 @@ describe("Repo", () => {
         eventPromise(client.networkSubsystem, "peer"),
       ])
 
-      const clientDoc = client.find(doc.url)
-      await pause(100)
-      assert.strictEqual(clientDoc.docSync(), undefined)
+      expect(async () => {
```

Comment:
When passing an async function to expect the return value becomes a promise itself which needs to be awaited.

```suggestion
      await expect(async () => {
```